app/src/main/java/swati4star/createpdf/util/TextToPdfAsync.java

```diff
@@ -39,9 +45,16 @@ protected void onPreExecute() {
         mOnPDFCreatedInterface.onPDFCreationStarted();
     }
 
+    private void createParentDir() {
```

Comment:
don't add utility fiunctions here. these should be in "FileUtils" or "DirectoryUtils"

Reply:
Removed createParentDir() function and started using DirectoryUtils function instead.